external/leveldb/LevelDbTuning.ipynb

```diff
@@ -175,499 +179,60 @@
-      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.splitter\": \"best\",\n",
-      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.max_depth\": 0,\n",
-      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_samples_split\": 2,\n",
-      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_samples_leaf\": 3,\n",
-      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_weight_fraction_leaf\": 0,\n",
-      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.max_features\": \"auto\",\n",
-      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.max_leaf_nodes\": 0,\n",
-      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_impurity_decrease\": 0,\n",
-      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.ccp_alpha\": 0,\n",
-      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.min_samples_to_fit\": 10,\n",
-      "  \"homogeneous_random_forest_regression_model_config.decision_tree_regression_model_config.n_new_samples_before_refit\": 1,\n",
-      "  \"homogeneous_random_forest_regression_model_config.bootstrap\": 1,\n",
-      "  \"experiment_designer_config.utility_function_implementation\": \"ConfidenceBoundUtilityFunction\",\n",
-      "  \"experiment_designer_config.numeric_optimizer_implementation\": \"RandomSearchOptimizer\",\n",
-      "  \"experiment_designer_config.confidence_bound_utility_function_config.utility_function_name\": \"upper_confidence_bound_on_improvement\",\n",
-      "  \"experiment_designer_config.confidence_bound_utility_function_config.alpha\": 0.1,\n",
-      "  \"experiment_designer_config.random_search_optimizer_config.num_samples_per_iteration\": 1000,\n",
-      "  \"experiment_designer_config.fraction_random_suggestions\": 0.5,\n",
-      "  \"experiment_designer_config_fraction_random_suggestions\": 0.1\n",
-      "}.\n"
-     ]
-    }
-   ],
+   "execution_count": 41,
+   "metadata": {},
+   "outputs": [],
    "source": [
-    "# Remember to initialize the optimizer before each time you run optimization or call run_optimizer() function\n",
-    "initialize_optimizer()"
+    "logger = create_logger('Optimizing LevelDB', logging_level=logging.WARN)\n",
+    "def initialize_optimizer():\n",
+    "    parameter_search_space = create_parameter_search_space(parameter_name=\"write_buffer_size\", min_val=1*1024*1024, max_val=128*1024*1024)\n",
+    "    optimization_problem = create_optimization_problem(parameter_search_space, objective_name=\"throughput\", min_val=0, max_val=1000, minimize=False)\n",
```

Comment:
Please add a comment that this `max_val` is assumed to be a excessive upperbound for the hardware the notebook was originally tested on, and may need be adjusted for the target platform in question.